src/treecontroller/model-converter-builder.js

```diff
@@ -0,0 +1,266 @@
+ * nodes which had converter attribute. It is suitable for attributes like `bold`, where `bold` attribute set on model
+ * text nodes is converter to `strong` view element.
+ *
+ *		BuildModelConverterFor( dispatcher ).fromAttribute( 'bold' ).toElement( 'strong' );
+ *
+ * It is possible to provide various different parameters for {@link engine.treeController.ModelConverterBuilder#toElement}
+ * and {@link engine.treeController.ModelConverterBuilder#toAttribute} methods. See their descriptions to learn more.
+ *
+ * It is also possible to {@link engine.treeController.ModelConverterBuilder#withPriority change default priority}
+ * of created converters to decide which converter should be fired earlier and which later. This is useful if you provide
+ * a general converter but want to provide different converter for a specific-case (i.e. given model element is converted
+ * always to given view element, but if it has given attribute it is converter to other view element). For this,
+ * use {@link engine.treeController.ModelConverterBuilder#withPriority withPriority} right after `from...` method.
+ *
+ * Note that `to...` methods are "terminators", which means that should be the last one used in building converter.
+ *
+ * You can use {@link engine.treeController.ViewConverterBuilder} to create "opposite" converters - from view to model.
+ *
+ * @memberOf engine.treeController
+ */
+class ModelConverterBuilder {
+	/**
+	 * Creates `ModelConverterBuilder` with given `dispatchers` registered to it.
+	 *
+	 * @param {Array.<engine.treeController.ModelConversionDispatcher>} dispatchers Dispatchers to which converters will
+	 * be attached.
+	 */
+	constructor( dispatchers ) {
+		/**
+		 * Dispatchers to which converters will be attached.
+		 *
+		 * @type {Array.<engine.treeController.ModelConversionDispatcher>}
+		 * @private
+		 */
+		this._dispatchers = dispatchers;
+
+		/**
+		 * Contains data about registered "from" query.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		this._from = null;
+	}
+
+	/**
+	 * Registers what model element should be converted.
+	 *
+	 * @chainable
+	 * @param {String} elementName Name of element to convert.
+	 * @returns {engine.treeController.ModelConverterBuilder}
+	 */
+	fromElement( elementName ) {
+		this._from = {
+			type: 'element',
+			name: elementName,
+			priority: null
+		};
+
+		return this;
+	}
+
+	/**
+	 * Registers what model attribute should be converted.
+	 *
+	 * @chainable
+	 * @param {String} key Key of attribute to convert.
+	 * @returns {engine.treeController.ModelConverterBuilder}
+	 */
+	fromAttribute( key ) {
+		this._from = {
+			type: 'attribute',
+			key: key,
+			priority: null
+		};
+
+		return this;
+	}
+
+	/**
+	 * Changes default priority for built converter.
+	 *
+	 * @param {Number} priority Converter priority.
+	 * @returns {engine.treeController.ModelConverterBuilder}
+	 */
+	withPriority( priority ) {
```

Comment:
Also this is both event priority and attribute priority, what should be mentioned.